### ISSUE TITLE
Possible to build and run without gtk (cli only)?

### ISSUE THREAD

**Issue by graysky2** [bleachbit/bleachbit#685]
I'd like to use bleachbit via a shell script without the gtk dependencies (400M of deps for gtk3).  Is there a way to build it without gtk?  If I attempt to run on without gtk:
```
scandir is not available, so falling back to slower os.walk()
Traceback (most recent call last):
  File "/usr/bin/bleachbit", line 42, in <module>
    import bleachbit.CLI
  File "/usr/share/bleachbit/CLI.py", line 27, in <module>
    from bleachbit.Cleaner import backends, create_simple_cleaner, register_cleaners
  File "/usr/share/bleachbit/Cleaner.py", line 42, in <module>
    from bleachbit.GuiBasic import Gtk, Gdk
  File "/usr/share/bleachbit/GuiBasic.py", line 29, in <module>
    gi.require_version('Gtk', '3.0')
  File "/usr/lib/python2.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace Gtk not available
```

**Comment by az0** [comment]
What command are you running? Try these: https://docs.bleachbit.org/doc/command-line-interface.html

**Comment by graysky2** [comment]
That as just running `/usr/bin/bleachbit --help`  According to the link you provided:
```
% /usr/bin/bleachbit --list
scandir is not available, so falling back to slower os.walk()
Traceback (most recent call last):
  File "/usr/bin/bleachbit", line 42, in <module>
    import bleachbit.CLI
  File "/usr/share/bleachbit/CLI.py", line 27, in <module>
    from bleachbit.Cleaner import backends, create_simple_cleaner, register_cleaners
  File "/usr/share/bleachbit/Cleaner.py", line 42, in <module>
    from bleachbit.GuiBasic import Gtk, Gdk
  File "/usr/share/bleachbit/GuiBasic.py", line 29, in <module>
    gi.require_version('Gtk', '3.0')
  File "/usr/lib/python2.7/site-packages/gi/__init__.py", line 133, in require_version
    (namespace, version))
ValueError: Namespace Gtk not available for version 3.0
```

This worked under v2.x

**Comment by az0** [comment]
@graysky2 
I made a change that should help. Could you please pull the new code from Git to test it, or you can try the [small patch](https://github.com/bleachbit/bleachbit/commit/3e3e49d323a228b857307d914fbb90180b4fbb93) to your existing installation.

**Comment by graysky2** [comment]
Nice, that patch fixed the issue.  Thank you!

**Comment by az0** [comment]
I published this fix in [BleachBit 3.1.0 beta](https://www.bleachbit.org/news/bleachbit-31-beta). Perhaps you could test and write back here with any comments about how it addresses your issue.

**Comment by graysky2** [comment]
> I published this fix in BleachBit 3.1.0 beta. Perhaps you could test and write back here with any comments about how it addresses your issue.

I downloaded the release you tagged at 3.1.0 and it builds/runs without the gtk3 deps or the patch.  Thank you.

**Comment by az0** [comment]
> > I published this fix in BleachBit 3.1.0 beta. Perhaps you could test and write back here with any comments about how it addresses your issue.
> 
> I downloaded the release you tagged at 3.1.0 and it builds/runs without the gtk3 deps or the patch. Thank you.

Thank you for confirming.
